src/Servers/IIS/IIS/test/testassets/InProcessWebSite/Program.cs

```diff
@@ -56,22 +56,26 @@ public static int Main(string[] args)
                 return 12;
             case "HangOnStop":
                 {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
This app explicitly targets 2.2 packages to make sure our IIS integration is backwards compatible.